wiki-webui/src/main/java/org/exoplatform/wiki/webui/control/action/ExportAsPDFActionComponent.java

```diff
@@ -144,8 +146,10 @@ private String htmlToxhtmlAndBase64Images(Page page) throws WikiException {
           }
           if(imageAttachment != null && imageAttachment.getMimeType() != null && imageAttachment.getMimeType().startsWith("image/")){
             byte[] bytes = imageAttachment.getContent();
+            ByteArrayInputStream buffArray = new ByteArrayInputStream(bytes);
+            BufferedImage buffImage = ImageIO.read(buffArray);
             element.attr("src", "base64," + Base64.encodeBase64String(bytes));
-            element.attr("style","width:100%;height:100%");
+            element.attr("style", "width:" + buffImage.getWidth() + "px;height:" + buffImage.getHeight() + "px;");
```

Comment:
By obtaining the same dimension of the supplied image. Will there be a risk in posting it (Display issues) ? (Consider the big pictures). Thanks

Reply:
@hbenali In WikiCKeditor there is no resize option, the image is inserted and rendered by its original size , we should obtain the same in the exported pdf file.
what display issues ? the defined dimension in this code is a temporary HTML file to the export the page

Reply:
@hakermi 

> what display issues? the defined dimension in this code is a temporary HTML file to the export the page

Consider a large image that has been put in the HTML file. when converting to pdf. We will not have an overflow effect? For example, will the image be cut off inside the page? I'm just curious. I will resolve the conversation. Thank you for your explanation.

Reply:
In your case, the code will be updated once they add the resize option in the Wiki, at that point we can obtain the chosen dimensions. but let's go on with you hypothesis, if the user add a big large image he will obtain the same display in the pdf file (a big large image) and yes it may be displayed in a new page in the pdf file (but this a normal behavior)